server/vulnerabilities/vuln_centos/centos.go

```diff
@@ -30,6 +30,10 @@ import (
 	_ "github.com/mattn/go-sqlite3"
 )
```

Comment:
Maybe can be considered a nit, but see https://github.com/fleetdm/fleet/pull/4037#discussion_r801647652